all-tools/check-tools

```diff
@@ -27,6 +27,7 @@ echo "--- Installed tools ---" \
     && echo "go-gofmt: $(go version)" \
     && echo "go-golint: ??" \
     && echo "go-megacheck: ??" \
+    && echo "go-staticcheck: ??" \
```

Comment:
There is a `-version` flag for `statickcheck`, you should call it here.

Reply:
Oh shoot -- thanks!

Reply:
I'm finding the behavior of the `-version` flag to be... perhaps inconsistent is the right word. On my local machine:
```
$ staticcheck -version
staticcheck (no version)
```

Reply:
So. It appears that the pre-built staticcheck binaries have versions, while installs from `go get` do not:
```
$ ./staticcheck_linux_amd64 -version
staticcheck_linux_amd64 2019.1
```
Do you prefer the Dockerfile pulls down a versioned binary, or do you prefer this use go-get and the version string remains `??`? I'm happy either way :+1:

Reply:
I'd rather use the `go-get` method if that's the preferred way to install it.  Too bad that it can't report its version number that way.  Let's leave it at `??` then.